jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -536,18 +537,20 @@ int jbmc_parse_optionst::doit()
 
   if(cmdline.args.empty())
   {
-    error() << "Please provide a program to verify" << eom;
+    log.error() << "Please provide a program to verify" << messaget::eom;
     return 6;
   }
 
   if(cmdline.args.size() != 1)
   {
-    error() << "Only one .class, .jar or .gbf file should be directly "
-      "specified on the command-line. To force loading another another class "
-      "use '--java-load-class somepackage.SomeClass' or "
-      "'--lazy-methods-extra-entry-point somepackage.SomeClass.method' along "
-      "with '--classpath'"
-            << messaget::eom;
+    log.error() << "Only one .class, .jar or .gbf file should be directly "
+                   "specified on the command-line. To force loading another "
+                   "another class "
```

Comment:
:pick: This could do with some manual re-wrapping to keep things readable and avoid overly-short lines.